docker/owlbot/java/bin/entrypoint.sh

```diff
@@ -15,33 +15,82 @@
 
 set -e
 
-# templates
-echo "Generating templates..."
-/owlbot/bin/write_templates.sh
-echo "...done"
-
-# write or restore pom.xml files
-echo "Generating missing pom.xml..."
-/owlbot/bin/write_missing_pom_files.sh
-echo "...done"
-
-# write or restore clirr-ignored-differences.xml
-echo "Generating clirr-ignored-differences.xml..."
-/owlbot/bin/write_clirr_ignore.sh
-echo "...done"
-
-# fix license headers
-echo "Fixing missing license headers..."
-/owlbot/bin/fix_license_headers.sh
-echo "...done"
-
-# TODO: re-enable this once we resolve thrashing
-# restore license headers years
-# echo "Restoring copyright years..."
-# /owlbot/bin/restore_license_headers.sh
-# echo "...done"
-
-# ensure formatting on all .java files in the repository
-echo "Reformatting source..."
-/owlbot/bin/format_source.sh
-echo "...done"
+# Runs template and etc in current working directory
+function processModule() {
+  # templates
+  echo "Generating templates..."
+  /owlbot/bin/write_templates.sh
+  echo "...done"
+
+  # write or restore pom.xml files
+  echo "Generating missing pom.xml..."
+  /owlbot/bin/write_missing_pom_files.sh
+  echo "...done"
+
+  # write or restore clirr-ignored-differences.xml
+  echo "Generating clirr-ignored-differences.xml..."
+  /owlbot/bin/write_clirr_ignore.sh
+  echo "...done"
+
+  # fix license headers
+  echo "Fixing missing license headers..."
+  /owlbot/bin/fix_license_headers.sh
+  echo "...done"
+
+  # TODO: re-enable this once we resolve thrashing
+  # restore license headers years
+  # echo "Restoring copyright years..."
+  # /owlbot/bin/restore_license_headers.sh
+  # echo "...done"
+
+  # ensure formatting on all .java files in the repository
+  echo "Reformatting source..."
+  /owlbot/bin/format_source.sh
+  echo "...done"
+}
+
+if [ "$(find . -type d -name 'java-*' |wc -l)" -gt 10 ];then
```

Comment:
I'd rather have something more robust that this heuristic. For example, it will fail if we're testing a monorepo with just a few modules. Would it make sense to find all of the directories with `owlbot.py` or `.OwlBot.yaml` and process from each of these directories. Then it wouldn't matter if it's a monorepo or not. Would this work?